### DISCUSSION TITLE
YouTube/Vimeo/Dailymotion Gameplay on Set Pages

### DISCUSSION THREAD

**Discussion by rabbids4eva** [RetroAchievements/RAWeb#737] [Ideas]
Usually, we see boxart and a screenshot for a game. Why not improve the game pages by providing gameplays on YouTube or other video sharing site to show the game in action?

**Comment by meleu** [comment]
Good idea. And easy to implement. :+1:

**Comment by SyrianBallaS** [comment]
I think this is good as well. GameFAQs has something similar but we should probably limit it to people playing with achievements on so potential players can see how it works first hand.
